network-api/networkapi/wagtailpages/pagemodels/customblocks/image_teaser_block.py

```diff
@@ -0,0 +1,46 @@
+from wagtail.core import blocks
+from wagtail.images.blocks import ImageChooserBlock
+from django.forms.utils import ErrorList
+from wagtail.core.blocks.struct_block import StructBlockValidationError
```

Comment:
What do you think about ordering the imports alphabetically? I often find that easier to navigate (I know most of the code base is not doing it... but I would like to make sure new code does)

Reply:
I've sorted these now and pushed up the change. 😄

Reply:
Definitely. It's on my list to set up. I just haven't done that yet because of the large integration branch we have going on. Once that is in I'll add black and isort.